ui.go

```diff
@@ -43,15 +42,37 @@ func newUI(w fyne.Window, p fyne.Preferences, host func() string) *ui {
 }
 
 func (u *ui) askShutdown() {
-	dialog.ShowConfirm("Shutdown", "Are you sure you want to shut down?",
-		func(ok bool) {
-			if !ok {
-				return
-			}
-
-			cmd := exec.Command("shutdown", "-h", "now")
-			_ = cmd.Start()
-		}, u.win)
+	var (
+		shutdown *widget.Button
+		reboot   *widget.Button
+		cancel   *widget.Button
+		pup      *widget.PopUp
+	)
```

Comment:
I'd prefer to remove the variable declarations here that are not needed. It is better visually, or perhaps more idiomatic Go, to just create the variables with `:=` instead and just keep the popup here because it is the only one that requires it.